src/site/content/en/blog/quictransport/index.md

```diff
@@ -0,0 +1,320 @@
+---
+title: Experimenting with QuicTransport
+subhead: QuicTransport is a new API offering low-latency, bi-directional, client-server messaging. Learn more about its use cases, and how to give feedback about the future of the implementation.
+authors:
+  - jeffposnick
+description: QuicTransport is a new API offering low-latency, bi-directional, client-server messaging. Learn more about its use cases, and how to give feedback about the future of the implementation.
+date: 2020-06-03
+updated: 2020-06-07
+hero: hero.jpg
+hero_position: center
+alt: |
+  Photo of fast-moving traffic.
+origin_trial:
+  url: https://developers.chrome.com/origintrials/#/view_trial/-6744140441987317759
+tags:
+  - blog
+  - capabilities
+  - network
+---
+
+## Background
+
+### What's QuicTransport?
+
+[QuicTransport](https://wicg.github.io/web-transport/#quic-transport) is a web API that uses the [QUIC](https://www.chromium.org/quic) protocol in a bidirectional, non-HTTP transport. It's intended for two-way communications between a web client and a QUIC server. It supports sending data both unreliably via its [datagram APIs](#datagram), and reliably via its [streams APIs](#stream).
```

Comment:
```suggestion
[QuicTransport](https://wicg.github.io/web-transport/#quic-transport) is a web API that uses the [QUIC](https://www.chromium.org/quic) protocol in a bi-directional, non-HTTP transport. It's intended for two-way communications between a web client and a QUIC server. It supports sending data both unreliably via its [datagram APIs](#datagram), and reliably via its [streams APIs](#stream).
```
Adds a hypen to `bi-directional` because that's how you formatted it in the front matter

Reply:
I am going to change the front matter in that case. The [specification](https://wicg.github.io/web-transport/#bidirectional-streams-transport ) uses "bidirectional" throughout, and there are API names that also use Bidirectional, so unless there's a strong web.dev editorial decision, I'll stick with no hyphen for everything.